src/Microsoft.DotNet.XHarness.iOS.Shared/IResultParser.cs

```diff
@@ -32,7 +32,7 @@ public interface IResultParser
 
         // parses the xml of the given jargon, create a human readable result and returns a result line with the summary of what was
         // parsed.
-        (string resultLine, bool failed) GenerateHumanReadableResults(string source, string destination, XmlResultJargon xmlType);
+        (string resultLine, bool failed) GenerateHumanReadableResults(string source, string destination, XmlResultJargon xmlType, bool generateHumanResults);
```

Comment:
I think this API is a bit confusing - I don't want to GenerateHumanReadableResults, so I just don't call this? 
Or why would I call it and then set false to the boolean? What would I expect to happen?

Reply:
just pass false to the boolean and it will not be generated, which is the default, maybe changing the name is a good idea

Reply:
What I am trying to say is:
if I am user of our library and I see `GenerateHumanReadableResults` I would just not call it at all. But that will cause problems, right?
I wouldn't call `GenerateHumanReadableResults(generateHumanResults: false)`

Also I was thinking this bool could be on a class level passed to the ctor as a behavior of the whole runner